src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,42 @@
+.Navbar {
+  width: 90%;
+  height: 50px;
+  margin: 0;
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  background-color: var(--color-white);
+  
+  &__left {
+    display: flex;
+    justify-content: flex-start;
+
+    .Navbar__icon img {
+      width: 150px;
+    }
+  }
+  
+  &__right {
+    flex: 3;
+    gap: 2em;
+    display: flex;
+    justify-content: flex-end;
+    
```

Comment:
please add 
 align-items: center;
 so that all links and button will be aligned